app/controllers/WKRemoteDataStoreController.scala

```diff
@@ -221,6 +221,22 @@ class WKRemoteDataStoreController @Inject()(
     }
   }
 
+  def updatePaths(name: String, key: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
+    dataStoreService.validateAccess(name, key) { _ =>
+      request.body.validate[List[DataSourcePathInfo]] match {
+        case JsSuccess(infos, _) =>
+          for {
+            _ <- datasetService.updateRealPaths(infos)(GlobalAccessContext)
+          } yield {
+            JsonOk
+          }
+        case e: JsError =>
+          logger.warn("Data store reported invalid json for data source paths.")
+          Fox.successful(JsonBadRequest(JsError.toJson(e)))
```

Comment:
Is there a specific reason to validate explicitly and log errors here, rather than using a playBodyParser to build an `Action[List[DataSourcePathInfo]]`?

Reply:
I did it for updateAll in https://github.com/scalableminds/webknossos/pull/8402 – but I’d say it’s not super important to update this now. Both is valid. The older has the benefit of controller-side logging (but I don’t think I ever saw this)

Reply:
Created https://github.com/scalableminds/webknossos/issues/8406 for that.